client/src/components/LeafletOsmMap/index.js

```diff
@@ -3,7 +3,8 @@ import style from "./style";
 import MapPane from './MapPane';
 import Search from '../../components/Search';
 import SearchResults from '../../components/SearchResults';
-import {fetchAndDropUserPins} from './SavedPlaces.js';
```

Comment:
Need to add this back...code doesn't work without this function

Reply:
I didn't realize it was deleted in the merge, I thought the other line was.